categories_map.py

```diff
@@ -1445,6 +1446,10 @@
     "SimG4CMS/HGCalTestBeam",
     "SimMuon/GEMDigitizer",
     "SimTracker/SiPhase2Digitizer",
+    "SimPPS/Configuration",
```

Comment:
@mundim why did you add these packages in the upgrade category? The upgrade category is specific to Phase2. I think these packages should just be placed in the general simulation category.